activerecord/lib/active_record/suppressor.rb

```diff
@@ -0,0 +1,57 @@
+module ActiveRecord
+  # ActiveRecord::Suppressor allows save events to be suppressed within a
```

Comment:
That's more along the lines of what I originally had and again included, but lost when I moved the comment up to the class level. There's a stray line break in there too, so I'll fix once there's feedback on this commit.

If you want you can blame the original author here: https://github.com/rails/rails/issues/18847. :smile: